src/components/ToggleSmall/ToggleSmall.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+const ToggleSmall = ({
+  className,
+  defaultToggled,
+  toggled,
+  onChange,
+  onToggle,
+  id,
+  ariaLabel,
+  ...other
+}) => {
+  let input;
+  const wrapperClasses = classNames({
+    'bx--form-item': true,
+    [className]: className,
+  });
+
+  const checkedProps = {};
+
+  if (typeof toggled !== 'undefined') {
+    checkedProps.checked = toggled;
+  } else {
+    checkedProps.defaultChecked = defaultToggled;
+  }
+
+  return (
+    <div className={wrapperClasses}>
+      <input
+        {...other}
+        {...checkedProps}
+        type="checkbox"
+        id={id}
+        className="bx--toggle bx--toggle--small"
+        onChange={evt => {
+          onChange && onChange(evt);
+          onToggle(input.checked, id, evt);
+        }}
+        ref={el => {
+          input = el;
+        }}
+        aria-label={ariaLabel}
+      />
+
+      <label className="bx--toggle__label" htmlFor={id}>
+        <span className="bx--toggle__appearance">
+          <svg
+            className="bx--toggle__check"
+            width="6px"
+            height="5px"
+            viewBox="0 0 6 5">
+            <path d="M2.2403 2.7299L4.9245 0 6 1.1117 2.2384 5 0 2.6863 1.0612 1.511z" />
+          </svg>
+        </span>
+      </label>
+    </div>
+  );
+};
+
+ToggleSmall.propTypes = {
+  className: PropTypes.string,
```

Comment:
Great to have comments explaining what those properties are for. If you do that, `Info` button at the top right will show those properties. The comment should be styled in:

```javascript
/**
 * `true` to make it toggled by default.
 */
defaultToggled: PropTypes.bool,
```

Reply:
Should we be adding a comment for every property moving forward, or just the properties we think need an explanation?

Reply:
Good question - I tend to comment everything just for a consistency check, but I see comments for some self-explanatory ones may be useless. My suggestion is being careful with ones that *look* self-explanatory and see if they really are (e.g. if we have `id` where is it applied to?). Thank you for asking!